server/odc-service/src/main/java/com/oceanbase/odc/service/iam/UserBatchImportPreviewer.java

```diff
@@ -120,33 +118,33 @@ private boolean checkFileType(MultipartFile file) {
     private BatchImportUser createBatchImportUser(Map<String, String> map, Set<String> accountNames,
             Map<String, Role> roleName2RoleMap) {
         BatchImportUser batchImportUser = new BatchImportUser();
-        String accountName = map.get(USER_ACCOUNTNAME.getLocalizedMessage());
-        if (!checkAccountName(accountName) || accountNames.contains(accountName)) {
-            batchImportUser.setErrorMessage("file content error:" + USER_ACCOUNTNAME.getLocalizedMessage());
-        } else {
-            batchImportUser.setAccountName(accountName);
-        }
+
         String name = map.get(USER_NAME.getLocalizedMessage());
-        if (StringUtils.isEmpty(name) || name.length() > NAME_MAX_LENGTH || !SPACE_PATTERN.matcher(name).matches()) {
-            batchImportUser.setErrorMessage("file content error:" + USER_NAME.getLocalizedMessage());
-        } else {
-            batchImportUser.setName(name);
-        }
+        Verify.verify(SPACE_PATTERN.matcher(name).matches(),
+                "username length should not exceed 64 and not start or end with spaces");
+        batchImportUser.setName(name);
+
+        String accountName = map.get(USER_ACCOUNTNAME.getLocalizedMessage());
+        Verify.verify(!accountNames.contains(accountName), "account name already exists");
+        Verify.verify(ACCOUNT_NAME_PATTERN.matcher(accountName).matches(),
+                "account name must be 4 to 64 characters long "
+                        + "and consist of letters, numbers, and special characters in ._+@#$% only");
```

Comment:
--> "Contains at least two lowercase letters, two uppercase letters, two numbers, and two special characters."